README.md

```diff
@@ -0,0 +1,23 @@
+# OpenTelemetry Operations Collector
+
+This repository is focused on building and packaging the OpenTelemetry Collector for use with Google Cloud Monitoring.
+
+## Prerequisite
+
+[docker](https://docs.docker.com/engine/install/) is installed
+
+## Building the Docker Image
+
+Run `make docker-build-image`
```

Comment:
```suggestion
```

Reply:
Do you think we don't need these here?

Reply:
I moved this section to the bottom in that other comment. You could leave the prerequisite section if you like.